.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: CI QA
+
+on:
+  push:
+  # pull_request:
+  #   branches:
+  #     - master
```

Comment:
Un-comment this?

Reply:
Yep, I'll add something else before uncomment